src/System.IO.FileSystem.DriveInfo/src/System.IO.FileSystem.DriveInfo.csproj

```diff
@@ -7,7 +7,7 @@
   <PropertyGroup>
     <RootNamespace>System.IO.FileSystem.DriveInfo</RootNamespace>
     <AssemblyName>System.IO.FileSystem.DriveInfo</AssemblyName>
-    <AssemblyVersion>4.0.0.0</AssemblyVersion>
+    <AssemblyVersion>4.0.0.1</AssemblyVersion>
```

Comment:
```xml
<AssemblyVersion Condition="'$(TargetGroup)' == 'net46'">4.0.0.0</AssemblyVersion>
```

Similar for the other files if they contain any TargetGroup which has a `IsPartialFacadeAssembly` value,